boards/nordic/thingy53/thingy53_nrf5340_common.dtsi

```diff
@@ -129,6 +129,13 @@
 				<&gpio1 10 0>,
 				<&gpio0 30 0>;
 		};
+
+		fem-spi-if {
+			gpios = <&gpio0 24 0>,
+				<&gpio0 29 0>,
+				<&gpio0 27 0>,
+				<&gpio0 28 0>;
```

Comment:
I've just noticed that these pins are also assigned to `spi3` which is enabled and has an enabled device (`adxl362`) on its bus: https://github.com/zephyrproject-rtos/zephyr/blob/ec0837e3474df47312664070139724e678046176/boards/nordic/thingy53/thingy53_nrf5340_common.dtsi#L227-L242
This won't work now since these pins will be assigned to the Network core even if it is not used (held in Force-OFF) and at least the [accel_polling](https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/sensor/accel_polling) seems to be using the accelerator. I guess both the nodes (`spi3` and `adxl362`) should be disabled by default then and an overlay should be added to the sample if the accelerator is to be still used there. Or maybe this SPI with sensors and FEM should be handled by the Application core (`nrf_radio_fem_spi` is also defined for `spi3`)? I'm not sure what was the original idea here.

Reply:
Application core can't handle FEM, the network core knows when the radio is in tx or rx mode, the application core does not. Seems this was not a good design

Reply:
Have reworked this to have 2 groups, SPI for FEM will not be set for network core but can be used if the commented dts entries are uncommented